pkg/build/packaging/grafana.go

```diff
@@ -444,11 +444,15 @@ func copyPubDir(grafanaDir, tmpDir string) error {
 	}
 
 	// Remove plugin backend code.
-	pluBeFiles, err := filepath.Glob(filepath.Join(tgtPubDir, "plugins", "*", "pkg"))
+	pluginFiles, err := filepath.Glob(filepath.Join(tgtPubDir, "plugins", "*", "*"))
 	if err != nil {
 		return err
 	}
-	for _, p := range pluBeFiles {
+	for _, p := range pluginFiles {
+		if filepath.Base(p) == "src" {
```

Comment:
Not sure this works with sub-directories of src directories?

Reply:
The `Glob` pattern only checks the main directory of the plugin, it doesn't go deeper than that:

![image](https://github.com/grafana/grafana/assets/4025665/9bbe5f91-f41a-45eb-a3d9-fb717d9408e4)

Reply:
Nice then you're good to go